CMakeLists.txt

```diff
@@ -75,8 +76,12 @@ add_subdirectory(src)
 if(UMF_BUILD_TESTS)
     add_subdirectory(test)
 endif()
-if(LINUX)
-    add_subdirectory(benchmark)
+if(UMF_BUILD_BENCHMARKS)
+    if(LINUX)
+        add_subdirectory(benchmark)
+    else()
+        message(FATAL_ERROR "benchmarks are supported on Linux only")
```

Comment:
What's the reason behind this? Is ubench only supported on linux?

Reply:
OS memory provider is not implemented for Windows.